TrafficSimulator/Assets/UI/Statistics/StatisticsController.cs

```diff
@@ -82,6 +82,14 @@ private void SettingsButtonOnClicked()
 
         private void UpdatePanels()
         {
+            Length MinHeight = new Length(25, LengthUnit.Pixel);
+            Length MaxHeight = new Length(30, LengthUnit.Pixel);
+            _statsButton.style.height = MinHeight;
+            _infoButton.style.height = MinHeight;
+            _settingsButton.style.height = MinHeight;
+            _statsButton.RemoveFromClassList("panel-button-selected");
+            _infoButton.RemoveFromClassList("panel-button-selected");
+            _settingsButton.RemoveFromClassList("panel-button-selected");
             switch(_tabs)
```

Comment:
This needs to be refactored. It has gotten too large so it looks very clunky explicitly disabling all other tabs and enabling the related one. The format is not consistent and it is easy to miss one toggle causing buggy behaviour.

It should be moved to a function that takes the tab as a parameter, then it can go through a list of all the panels and disable all except the one in the parameter, and enable that one. This can also be done for the buttons although that might be more difficult if they do not share a common type/parent class